gammapy/data/data_store.py

```diff
@@ -209,6 +209,25 @@ def load(self, obs_id, filetype):
         else:
             raise ValueError('Invalid filetype.')
 
+    def load_all(self, filetype):
+        """Load data file as appropriate object for a
+
+        Parameters
+        ----------
+        filetype : {'events', 'aeff', 'edisp', 'psf', 'background'}
+            Type of file.
+
+        Returns
+        -------
+        list : python list of object
+            Object depends on type, e.g. for `events` it is a list of `~gammapy.data.EventList`.
+        """
+        event_lists = []
```

Comment:
Rename `event_lists` -> `data_list`.

(I can't think of a good name here ... my main point is to not use `event_lists` since something this will be a list of effective area or other objects.)